app/src/main/java/com/plenart/organizeme/adapters/TaskListItemsAdapter.kt

```diff
@@ -55,16 +53,16 @@ class TaskListItemsAdapter(
         val builder = AlertDialog.Builder(context)
 
         builder.apply {
```

Comment:
Better practise is to have some delete listener (like for click listener) in adapter for deleting item and then in view (fragment) do the logic for showing dialog. 

P.S. same is for showing toasts, but for toasts there is no need for listeners because here you have just logic for example validating user input or something like that and you can do that in view with help of ViewModel.